lib/restforce/db/associations/belongs_to.rb

```diff
@@ -45,6 +45,31 @@ def build(database_record, salesforce_record, cache = AssociationCache.new)
           @cache = nil
         end
 
+        # Public: Get a Hash of Lookup IDs for a specified database record,
+        # based on the current record for this association.
+        #
+        # database_record - An instance of an ActiveRecord::Base subclass.
+        #
+        # Returns a Hash.
+        def lookups(database_record)
+          ids = {}
+
+          for_mappings(database_record) do |mapping, lookup|
+            associated = database_record.association(name).reader
+
+            if associated
+              # It's possible to define a belongs_to association in a Mapping
+              # for what is actually a one-to-many association in ActiveRecord.
+              associated = associated.first if associated.respond_to?(:first)
```

Comment:
Good catch; I didn't originally have the nil check in place, and didn't refactor when I added it.